frontend/src/app/components/wp-tabs/components/wp-tabs/wp-tabs.component.html

```diff
@@ -1,5 +1,68 @@
-<li *ngFor="let tab of tabs" [uiSref]="uiSref"
-    [uiParams]="{workPackageId: workPackage.id, tabIdentifier: tab.identifier}"
-    uiSrefActive="selected">
-  <a href="" [textContent]="tab.displayName"></a>
-</li>
+<ul class="op-work-package-tabs tabrow">
+  <li
+      *ngIf="view === 'split'"
+      uiSref="{{ uiSrefBase }}.overview"
+      [uiParams]="{workPackageId: workPackage.id}"
+      uiSrefActive="selected"
+  >
+    <a href="" [textContent]="text.tabs.overview"></a>
```

Comment:
Aren't the uiSref and uiParams on the wrong element here?

Reply:
The problem here is the style coming from a generic "tabrow" style component, which styles the `li` to have a padding. Placing uiSref on the link then would not trigger on a click outside of the actual text.

Ideally we also change the styling of the `tabrow` but I'm not sure that this should be in the scope of this PR